src/docs/software/using/R.md

```diff
@@ -439,7 +439,7 @@ $ sh_dev -c 4
 
 $ ml purge
 $ ml R/4.2.0
-$ ml system harfbuzz fribidi
+$ ml system harfbuzz fribidi libwebp
```

Comment:
At this point, given your recent experience, maybe it would be even better to completely specify the module versions for each dependency, to make sure things are reproducible?